include/net/nrf_cloud.rst

```diff
@@ -89,6 +89,33 @@ When the device is successfully associated with a user on the cloud, subsequent
 
 After receiving :c:enumerator:`NRF_CLOUD_EVT_READY`, the application can start sending sensor data to the cloud.
 
+.. _lib_nrf_cloud_fota:
+
+Firmware Over-the-Air (FOTA) Updates
+************************************
+nRF Connect for Cloud supports device FOTA updates.  :option:`NRF_CLOUD_FOTA` is enabled by default when using this nRF Cloud library.
+The device must inform nRF Connect for Clound of which FOTA type(s) it supports.
+This is done by writing a "fota_v2" field containing an array of FOTA types into the "serviceInfo" field in the device's shadow.
+The three supported FOTA types are: "APP", "MODEM", and "BOOT"
+For example, a device that supports all three FOTA types would write the following data to its shadow:
+
+``{
+"state": {
+   "reported": {
+      "device": {
+         "serviceInfo": {
+            "fota_v2": [
+            "APP",
+            "MODEM",
+            "BOOT"
+            ]
+}}}}}``
+
```

Comment:
Would it make sense to refer to a specific part of asset_tracker as a reference of how to do this?  Same for handling the events below?

Reply:
i'm not sure it is "correct" for library documentation to reference an application or sample project.  what are your thoughts on this @umapraseeda ?  
i feel that the cloud codec stuff done in asset tracker needs to be moved into the nrf cloud lib, since it is quite cloud specific.

Reply:
Added a small section in Asset Tracker to link to this documentation based on offline discussion.